cylc/doc/etc/tutorial/cylc-forecasting-suite/bin/consolidate-observations

```diff
@@ -1,9 +1,21 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 # -----------------------------------------------------------------------------
-# Copyright (C) 2012-2019 British Crown (Met Office) & Contributors - GNU V3+.
-# This is illustrative code developed for tutorial purposes, it is not
-# intended for scientific use and is not guarantied to be accurate or correct.
```

Comment:
I think this was put here for a reason, best to keep.

Reply:
Addressed in https://github.com/cylc/cylc-doc/pull/186/commits/77d3efb90e0ba2df34c1f0c058dfe73b6643073c